src/pgzero/music.py

```diff
@@ -69,7 +69,7 @@ def queue(name):
     _music.queue(path)
 
 
-def is_playing(name):
+def is_playing():
```

Comment:
Unfortunately this shipped in Pygame Zero 1.2 etc so it is backwards incompatible to just remove it. We need to just make it an optional parameter.

```suggestion
def is_playing(name=None):
```